### DISCUSSION TITLE
Cloud <-> Self-hosted migration

### DISCUSSION THREAD

**Discussion by andrasbacsai** [coollabsio/coolify#3176] [Feature Requests]
Able to migrate in both directions.

**Comment by sawirricardo** [comment]
Hi, any update on this? I'm looking for self hosted Coolify to Coolify Cloud

**Reply by watate** [reply]
Same here, would love to migrate and pay $$$ to support you @andrasbacsai and also remove a lot of the maintenance headaches!

**Comment by RVP97** [comment]
Hey, are there updates on this?

**Comment by sovetski** [comment]
any updates here?
